homework-submission/week2/src/App.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import './App.css';
+import TodoList from './todo/TodoList';
+import todos from './todo/todos';
+
+class App extends Component {
+  state = {
+    todos,
+  };
+
+  handelChange = id => {
+    const newTodos = this.state.todos;
+    newTodos.map(todo => {
```

Comment:
line 12: don't mutate the state directly make a deep copy, important

Reply:
Thanks @SarKurd for your review 👍